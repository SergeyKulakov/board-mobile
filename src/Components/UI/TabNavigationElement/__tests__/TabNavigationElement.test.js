import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { TabNavigationElement as Component } from '../index'

describe('TabNavigationElement', () => {
  const defaultProps = {
    isActive: false,
    isLoading: false,
    onClick: () => null,
    title: 'title',
    style: undefined,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)
    const witLoading = shallow(<Component {...defaultProps} isLoading />)

    expect([wrapper, witLoading]).toMatchSnapshot()
  })
})
