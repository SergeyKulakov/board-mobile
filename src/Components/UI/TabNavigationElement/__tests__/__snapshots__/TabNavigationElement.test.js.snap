// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabNavigationElement render snapshot 1`] = `
Array [
  <Styled(TouchableOpacity)
    onPress={[Function]}
  >
    <Styled(Text)
      active={false}
    >
      title
    </Styled(Text)>
    <Styled(View)
      active={false}
    />
  </Styled(TouchableOpacity)>,
  <Styled(TouchableOpacity)
    onPress={[Function]}
  >
    <Styled(ActivityIndicator)
      animating={true}
      color="#999999"
      hidesWhenStopped={true}
      size="small"
    />
    <Styled(View)
      active={false}
    />
  </Styled(TouchableOpacity)>,
]
`;
